internal/cli/client/token_refresher.go

```diff
@@ -50,19 +50,24 @@ func (t *tokenRefresher) refreshToken(
 	insecureTLS bool,
 ) (config.CLIConfig, error) {
 	jwtParser := jwt.NewParser(jwt.WithoutClaimsValidation())
-	var claims jwt.RegisteredClaims
+	var untrustedClaims jwt.RegisteredClaims
```

Comment:
Changing the name of this symbol is technically unrelated to this change, but while I was already touching this file, I thought it worthwhile to make it clear that these claims aren't trusted (since the signature isn't verified).